stdlib/Random/docs/src/index.md

```diff
@@ -66,6 +71,18 @@ Random.MersenneTwister
 Random.RandomDevice
 ```
 
+## Cryptographically secure random numbers
+
+The Julia standard library does not provide a native cryptographically secure random number generator. For most applications,
+we recommend to use the operating system `RandomDevice()`. This should be initialized via a module-level
+`const SECURE_RANDOM = Random.RandomDevice()` and subsequently used via e.g. `gen_nonce() = rand(SECURE_RANDOM, UInt128)`.
```

Comment:
Only by the name `SECURE_RANDOM` we can infer that indeed `RandomDevice` is secure, it should be stated explicitly.